tests/utils/tools.php

```diff
@@ -137,6 +137,25 @@ function LOAD($uri, $dbname = DATABASE_NAME, $collname = COLLECTION_NAME, $filen
     }
 }
 
+function NEEDS_ATLEAST_MONGODB_VERSION($uri, $version) {
+    $manager = new MongoDB\Driver\Manager($uri);
+    $cmd = new MongoDB\Driver\Command(array("buildInfo" => 1));
```

Comment:
Short array syntax?

Reply:
I can change that, if you really care :-)

Reply:
Please do while merging. It'll just save time later if we come across it again and want to revise.